pkg/base/k8s/options.go

```diff
@@ -0,0 +1,28 @@
+package k8s
+
+var DefaultMaxBackoffRetries = 0
+
+type KubeClientRequestOptions struct {
```

Comment:
Can we have shorter naming ? `ClientOptions` instead of `KubeClientRequestOptions` for example or `DefaultOptions` for `DefaultKubeClientRequestOptions`. It's hard to read `Java` style naming in `go`.  Please use short, concise, evocative naming.

Reply:
Renamed options